ipatests/test_integration/util.py

```diff
@@ -17,51 +17,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import time
 import re
 
 from ipaplatform.paths import paths
 from ipalib.constants import DEFAULT_CONFIG
 
```

Comment:
```base.py``` provides base class for all integration tests. The pytest plugin doesn't depend on the base class directly IMO it makes more sense to keep it next to the test cases.

For the other functions in module, I don't really care. If you prefer to have them in tasks, I'm happy to move them, too.